app/services/proofing/aamva/proofer_factory.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module Proofing
+  module Aamva
+    class ProoferFactory
+      attr_accessor :resolution_context
+
+      # @param [Proofing::Resolution::ResolutionContext] resolution_context
+      def initialize(resolution_context)
+        @resolution_context = resolution_context
+      end
```

Comment:
It's unnecessary to keep `resolution_context` in the factory state. None of the usages take additional actions before creating the proofer objects. (applies to both proofers)